### PR TITLE
Apply welsh translations to lms

### PR DESCRIPTION
**Pull request by dcdarrell9**
### What is the context of this PR?
- Added lms_2 welsh schema

### How to review 
- Needs to be thoroughly tested 
- Make sure all routes are working
- No errors with piped values
- Test switching between languages
- Try with static content as well

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by dcdarrell9** [comment]
** Need to go through and re-add smart quotes in welsh words **

File: data/cy/lms_2.json [hunk 1]
```diff
@@ -0,0 +1,8043 @@
...
+                                }
+                            ],
+                            "type": "General",
+                            "answers": [{
+                                    "id": "country-of-birth-wales-answer",
+                                    "mandatory": false,
+                                    "type": "Radio",
+                                    "options": [{
+                                            "label": "Cymru",
+                                            "value": "
```
Comment (dcdarrell9) [inline data/cy/lms_2.json]: `Optional start and finish times` && `Job Sharing` wrong in translation file

Optional start and finish times - "Amseroedd cychwyn a gorffen dewisol"
Job Sharing - "Rhannu swydd"

Comment (dcdarrell9) [reply data/cy/lms_2.json]: This is pointing to line 1121 for some reason, the label is on line 3480

**Review by bitdivision — APPROVED** [review]

File: data/cy/lms_2.json [hunk 2]
```diff
@@ -0,0 +1,16226 @@
...
+                            "type": "Relationship",
+                            "member_label": "answers['primary-household-member-first-name'] | default(answers['other-household-member-first-name']) | default(answers['student-household-member-first-name']) | default(answers['other-household-member-first-name-no-primary']) | default(answers['student-household-member-first-name-no-primary'])",
+                            "answers": [{
+                                "id": "no-primary-who-is-related",
+                                "label": "%(current_person)s yw… &hellip; %(other_person)s",
```
Comment (MebinAbraham) [inline data/cy/lms_2.json]: As discussed, extra random  `...`

Comment (ajmaddaford) [reply data/cy/lms_2.json]: Fixed

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**